README.md

```diff
@@ -5,6 +5,12 @@
   <img src="https://img.shields.io/npm/dt/@dropbox/ttvc"/>
   <img src="https://img.shields.io/bundlephobia/minzip/@dropbox/ttvc"/>
   <img src="https://img.shields.io/bundlephobia/min/@dropbox/ttvc"/>
+  <img src="https://img.shields.io/github/checks-status/dropbox/ttvc/main"/>
+  <img src="https://img.shields.io/github/checks-status/dropbox/ttvc/main"/>
+  <img src="https://img.shields.io/github/issues-raw/dropbox/ttvc"/>
+  <img src="https://img.shields.io/github/issues-closed-raw/dropbox/ttvc"/>
+  <img src="https://img.shields.io/github/license/dropbox/ttvc"/>
+  <img src="https://img.shields.io/github/commit-activity/m/dropbox/ttvc"/>
```

Comment:
It is neat that all of these are possible, but I feel some of them might be distracting from the interesting information, haha.  😅 

(Also, I'm not sure highlighting the very limited activity we've had so far is super helpful!  😉)

I think I'd rather remove these... for now.  Are you okay with that?